.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python: 2.7
+sudo: false
+cache:
+  directories:
+  - eggs
+  - parts/node
+env:
+  matrix:
+    - PLONE_VERSION=4.3
+    - PLONE_VERSION=5.0
+    - PLONE_VERSION=5.1
+install:
+- sed -ie "s#test-4.3#test-$PLONE_VERSION#" buildout.cfg
+- python bootstrap-buildout.py
```

Comment:
@tomgross I usually switch to pip install zc.buildout here :-) see https://github.com/plone/plone.app.tiles/blob/master/.travis.yml